app/src/main/java/de/westnordost/streetcomplete/quests/AbstractQuestAnswerFragment.kt

```diff
@@ -275,6 +276,15 @@ abstract class AbstractQuestAnswerFragment<T> : AbstractBottomSheetFragment() {
         const val ARG_MAP_ROTATION = "map_rotation"
         const val ARG_MAP_TILT = "map_tilt"
     }
+
+
+    fun showHint(@StringRes messageId: Int){
+        activity?.let { AlertDialog.Builder(it)
+                .setMessage(messageId)
+                .setPositiveButton(android.R.string.ok, null)
+                .show()
+        }
+    }
```

Comment:
Good idea to reuse this snippet. However, showing a hint is nothing specific to AbstractQuestAnswerFragment.

Better create an extension function on `Context` and put it somewhere in the ktx package directory.

```
fun Context.showHint(@StringRes messageId: Int){
   AlertDialog.Builder(this)
        .setMessage(messageId)
        .setPositiveButton(android.R.string.ok, null)
        .show()
}
```
And use like `activity?.showHint(...)`